truffle/src/main/java/org/jruby/truffle/RubyEngine.java

```diff
@@ -35,6 +36,7 @@ public RubyEngine(RubyInstanceConfig instanceConfig) {
 
         engine = PolyglotEngine.newBuilder()
                 .config(RubyLanguage.MIME_TYPE, INSTANCE_CONFIG_KEY, instanceConfig)
+                .config(RubyLanguage.MIME_TYPE, OptionsCatalogue.ARGUMENTS.getName(), instanceConfig.getArgv())
```

Comment:
How about passing them as a HashMap or so instead?
This could become quickly a long list, isn't it?